pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -1965,7 +1966,7 @@ func convertGateways(r configContext) ([]config.Config, map[parentKey][]*parentI
 	// namespaceLabelReferences keeps track of all namespace label keys referenced by Gateways. This is
 	// used to ensure we handle namespace updates for those keys.
 	namespaceLabelReferences := sets.New[string]()
-	classes := getGatewayClasses(r.GatewayResources)
+	classes := getGatewayClasses(r.GatewayResources, getSupportedFeatures())
```

Comment:
And i think we donot have to calculate it each time, we can  declare a variable.

Reply:
thanks, changed